React/Views/RCTSlider.m

```diff
@@ -32,4 +32,17 @@ - (void)setMaximumValue:(float)maximumValue
   super.value = _unclippedValue;
 }
 
+-(void)setTrackImage: (NSString *) trackImageSrc
+{
+
+  UIImage *sliderTrackImage = [UIImage imageNamed: trackImageSrc];
+  NSInteger width = sliderTrackImage.size.width;
+
+  UIImage *sliderLeftTrackImage = [sliderTrackImage resizableImageWithCapInsets:UIEdgeInsetsMake(0, width, 0, 0)];
+  UIImage *setMaximumTrackImage = [sliderTrackImage resizableImageWithCapInsets:UIEdgeInsetsMake(0, 0, 0, width)];
```

Comment:
It depends on the result you want to achieve. If your goal is to have a simple track image, I would say this is the best way.  If we make these separate properties, it will give more flexibility on what can be achieved, but it will make having a simple track image more complicated